src/writer/serve.py

```
@@ -379,8 +379,30 @@ async def get_blueprints(request: Request):
+                "cron_expression": comp.get("content", {}).get("cronExpression", ""),
+                "timezone": comp.get("content", {}).get("timezone", "UTC"),
+            }
+            for comp in app_runner.bmc_components.values()
+            if comp.get("type") == "blueprints_crontrigger"
+        ]
+
+        return JSONResponse(content=cron_triggers, status_code=200)
+
     @app.post("/private/api/blueprint/{blueprint_id}")
-    async def create_blueprint_job(blueprint_id: str, request: Request, response: Response):
+    async def create_blueprint_job(blueprint_id: str, request: Request, response: Response, branch_id: str | None = None):
```

Ruff (0.14.4)

Unused function argument: `response`

(ARG001)